addons/survey/static/src/js/survey_form.js

```diff
@@ -537,6 +543,74 @@ publicWidget.registry.SurveyFormWidget = publicWidget.Widget.extend(SurveyPreloa
         }
     },
 
+    _showCorrectAnswers: function(correctAnswers, submitPromise, options) {
```

Comment:
Can I be "annoying" and ask you to move these with the "showX"/"toggleX"? methods instead of in the handlers :pleading_face: ?